cmd/proxy/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM ubuntu:bionic
+WORKDIR /
+COPY manager.exe ./manager
+
+ENTRYPOINT ["/manager"]
```

Comment:
it's not a `manager`. we can use a different name like `proxy`

Reply:
or `proxy-app`

Reply:
change to proxy-app